src/inverseRobot.js

```diff
@@ -24,7 +24,28 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const mainPropArray = [];
+  const robotReverse = {};
+
+  for (const key in robot) {
+    mainPropArray.push(key, robot[key]);
+  }
+
+  mainPropArray.sort();
+
+  for (let i = 0; i < mainPropArray.length; i++) {
+    if (mainPropArray[i] === mainPropArray[i + 1]) {
+      return null;
+    }
+  }
```

Comment:
There are much more easier ways to check if an object has certain property.
Use `hasOwnProperty` here.